README.md

```diff
@@ -0,0 +1,83 @@
+# Unikraft Website
+
+Contents from this repository are published at https://unikraft.org/ with sub-links available from the top bar:
+
+- documentation: https://unikraft.org/docs/
+- community information (including hackathons): https://unikraft.org/community/
+- blog: https://unikraft.org/blog/
+
+The Unikraft website is written in [Markdown](https://www.markdownguide.org/) and published via [Hugo](https://gohugo.io/) and the [Docsy theme](https://www.docsy.dev/).
+
+## Build
+
+You can build the website inside a Docker environment.
+For this, you will need to install Docker-CE.
+You can do that by following the instructions [here](https://github.com/docker/docker-install), or simply run:
+
+**Note: This is highly recomended, since you will work in the same environment as the deployed website, so there will be no errors regarding packages version or filesystem layout.**
+
+```bash
+curl -fsSL https://get.docker.com/ | sh
+```
+
+To then deploy the website, run:
+
+```bash
+$ make container
+$ make devenv
+
+# To be run inside the docker env
+$ npm install
+$ make serve
+```
+
+You may need special privileges to run the first 2 commands.
+
+You can also build the website natively.
+You will need to install `hugo`.
+You can do it by following the instructions [here](https://gohugo.io/getting-started/installing/).
+
+Then, to build and deploy the website, run:
+
+```bash
+$ hugo serve
+```
+
+**Please note that this may lead to errors due to different hugo versions or other environment problems.
```

Comment:
```suggestion
**Note: This may lead to errors due to different `hugo` versions or other environment problems.
```
Changed this in order to match the other "Note:" that we have in the document.